### PR TITLE
NV-2469 - 🚀 Feature: Rename Sendinblue to Brevo

### PR DESCRIPTION
**Pull request by gitstart**
### What change does this PR introduce?
This PR implement the changes needed to support the rebranding of Sendinblue to Brevo, this includes to the UI or both the web app and the documentation

### Why was this change needed?
Closes: https://github.com/novuhq/novu/issues/3627

### Other information

[Loom Video](https://www.loom.com/share/0251e1100cb04f33be3f8f613fabcf8d?sid=adc5781c-02c5-4358-a81f-97aa1db2b974)

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
@p-fernandez this PR is ready for review

File: libs/shared/src/consts/providers/channels/email.ts [hunk 1]
```diff
@@ -62,12 +62,12 @@ export const emailProviders: IProviderConfig[] = [
     logoFileName: { light: 'sendgrid.png', dark: 'sendgrid.png' },
   },
   {
-    id: EmailProviderIdEnum.Sendinblue,
-    displayName: 'Sendinblue',
+    id: EmailProviderIdEnum.Brevo,
```
Comment (scopsy) [inline libs/shared/src/consts/providers/channels/email.ts:65]: Unfortuently this change is not backwards compatible for scheduled jobs using `sendinblue` and they will just fail. My suggestion is to create backwards compatible approach where both enums still exist, and during the handler selection we process both sendinblue and brevo using the same handler.

**Review by scopsy — CHANGES_REQUESTED** [review]

File: libs/shared/src/consts/providers/channels/email.ts (hunk 1 above)
Comment (gitstart) [reply libs/shared/src/consts/providers/channels/email.ts]: Your feedback has been implemented @scopsy, thanks for the review.

**Comment by scopsy** [comment]
Unfortunately there are still a few problems here :( Will try to comment on the relevant places a bit later 🙏

**Comment by gitstart** [comment]
Noted. Eagerly waiting for your review 🙏

**Pull request closed without merging**